### PR TITLE
feat: add facebook share button

### PR DESCRIPTION
**Pull request by Rabinzon**
Fixes issue #200 

Changes: Facebook share button added 

Screenshots for the change: http://take.ms/b8oQh  http://take.ms/eAmkj

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -38,4 +38,11 @@ h1 {
...
-}
\ No newline at end of file
+}
+
+.social-buttons {
+  display: flex;
+  width: 180px;
+  margin: auto;
+  justify-content: space-around;
+}
```
Comment (jenkoian) [inline public/css/main.css]: Rather than add a new class, can we use tachyons utility classes? http://tachyons.io/

File: public/js/main.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ HacktoberfestChecker.prototype.getUsernameIssues = function(e) {
 HacktoberfestChecker.prototype.usernameIssuesSuccess = function(html, textStatus, xhr) {
     this.results.html(html);
     window.twttr.widgets.load();
-
+    window.FB.XFBML.parse();
```
Comment (jenkoian) [inline public/js/main.js]: Can we add a newline after. Alos may be nice to put this and the twitter one in a `loadSocialWidgets()` method or something?

Comment (Rabinzon) [reply public/js/main.js]: sure, one moment)

**Comment by jenkoian** [comment]
Thanks @Rabinzon !

**Pull request merged**